packages/attendance/src/components/AttendanceComponent.js

```diff
@@ -901,7 +902,7 @@ const CalendarComponent = ({
             key={subIndex}
             alignItems="center"
             borderWidth={isToday ? "1" : ""}
-            borderColor={isToday ? "button.500" : ""}
+            borderColor={isToday ? "#BCBCBC" : ""}
```

Comment:
Create a color object and name all of these colors. We should move towards creating a theme.